From 68fcab9ae6f5c7e3ca40be8991138fa173a08fbd Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@st.com>
Date: Wed, 26 Aug 2020 15:34:53 +0200
Subject: [PATCH 41/65] spi: stm32_qspi: Fix AARCH64 compilation warnings

When building with AARCH64 defconfig, we got warning, fix it.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/u-boot.git c9e25ec840e9f5a9053c0027d8681a3214230b37 spi: stm32_qspi: Fix AARCH64 compilation warnings]
Signed-off-by: Patrice Chotard <patrice.chotard@st.com>
Change-Id: Ic78f420e29405a4c04e5e85b61dc422e03612c22
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/176974
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
Reviewed-by: CIBUILD <smet-aci-builds@lists.codex.cro.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
---
 drivers/spi/stm32_qspi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/spi/stm32_qspi.c b/drivers/spi/stm32_qspi.c
index 3216ec8010e..612acf9e509 100644
--- a/drivers/spi/stm32_qspi.c
+++ b/drivers/spi/stm32_qspi.c
@@ -346,8 +346,8 @@ static int stm32_qspi_probe(struct udevice *bus)
 	if (priv->mm_size > STM32_QSPI_MAX_MMAP_SZ)
 		return -EINVAL;
 
-	dev_dbg(bus, "regs=<0x%p> mapped=<0x%p> mapped_size=<0x%lx>\n",
-		priv->regs, priv->mm_base, priv->mm_size);
+	dev_dbg(bus, "%s: regs=<0x%p> mapped=<0x%p> mapped_size=<0x%x>\n",
+		__func__, priv->regs, priv->mm_base, (u32)priv->mm_size);
 
 	ret = clk_get_by_index(bus, 0, &clk);
 	if (ret < 0)
-- 
2.51.0

